securesignals-ima-devapp/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <application
```

Comment:
Nit: If possible, let's be consistent with the ordering and definition between the two dev apps. If you compare the other `AndroidManifest`, a few things pop out:
 - Label should be first
 - Label names: "UID2 SDK Dev App" vs "SecureSignalsIMADevApp" (consider renaming to "UID2 IMA Dev App"?)
 - This Dev App uses the internet, but missing the required `uses-permission`.

Reply:
Done.